scheduler/src/cook/rest/api.clj

```diff
@@ -1968,10 +1969,14 @@
   ::jobs and ::groups, which specify the jobs and job groups."
   [conn {:keys [::groups ::job-pool-name-maps] :as ctx}]
   (try
-    (log/info "Submitting jobs through raw api:"
-              (map (fn [job-pool-name-map]
-                     (update job-pool-name-map :job #(dissoc % :command)))
-                   job-pool-name-maps))
+    (let [jobs (map (fn [job-pool-name-map]
+                      (update job-pool-name-map :job #(dissoc % :command)))
+                    job-pool-name-maps)]
+      (log/info "Submitting jobs through raw api:" jobs)
+      (doseq [{{:keys [uuid, user]} :job} jobs]
+        (passport/log-event {:event-type passport/api-job-submission
+                             :job-uuid (str uuid)
+                             :user user})))
```

Comment:
How about this, to avoid the extra `let`?

```suggestion
    (log/info "Submitting jobs through raw api:"
              (map (fn [job-pool-name-map]
                     (update job-pool-name-map :job #(dissoc % :command)))
                   job-pool-name-maps))
    (doseq [{{:keys [uuid, user]} :job} job-pool-name-maps]
      (passport/log-event {:event-type passport/api-job-submission
                           :job-uuid (str uuid)
                           :user user}))
```

Reply:
Sure, this works for me. I'll add & test it then post another revision